src/index.html

```diff
@@ -1,13 +1,69 @@
 <!doctype html>
-<html lang="en">
-  <head>
+<html lang="en" class="page">
+  <head class="page__head">
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="page__body">
+    <header class="header"></header>
+    
+    <main class="main">
+      <!-- #create stars -->
+      <div class="main__block">
+        <div class="stars stars--0">
+          <div href="#" class="stars__star"></div>
+          <div href="#" class="stars__star"></div>
+          <div href="#" class="stars__star"></div>
+          <div href="#" class="stars__star"></div>
+          <div href="#" class="stars__star"></div>
```

Comment:
```suggestion
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
```
div hasn't attribute href and brawser anyway ignore it